content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+{
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669297890, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
+      keyId: Long("0")
+    }
+  },
+  operationTime: Timestamp({ t: 1669297890, i: 1 })
+}
+
+Validate that mongo-0 pod is registered as Primary Replica
+
+rs0 [direct: other] test> rs.status()
+{
+  set: 'rs0',
+  date: ISODate("2022-11-24T13:52:34.600Z"),
+  myState: 1,
+  term: Long("1"),
+  syncSourceHost: '',
+  syncSourceId: -1,
+  heartbeatIntervalMillis: Long("2000"),
+  majorityVoteCount: 1,
+  writeMajorityCount: 1,
+  votingMembersCount: 1,
+  writableVotingMembersCount: 1,
+  optimes: {
+    lastCommittedOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    lastCommittedWallTime: ISODate("2022-11-24T13:52:30.843Z"),
+    readConcernMajorityOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    appliedOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    durableOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    lastAppliedWallTime: ISODate("2022-11-24T13:52:30.843Z"),
+    lastDurableWallTime: ISODate("2022-11-24T13:52:30.843Z")
+  },
+  lastStableRecoveryTimestamp: Timestamp({ t: 1669297900, i: 1 }),
+  electionCandidateMetrics: {
+    lastElectionReason: 'electionTimeout',
+    lastElectionDate: ISODate("2022-11-24T13:49:50.807Z"),
+    electionTerm: Long("1"),
+    lastCommittedOpTimeAtElection: { ts: Timestamp({ t: 1669297790, i: 1 }), t: Long("-1") },
+    lastSeenOpTimeAtElection: { ts: Timestamp({ t: 1669297790, i: 1 }), t: Long("-1") },
+    numVotesNeeded: 1,
+    priorityAtElection: 1,
+    electionTimeoutMillis: Long("10000"),
+    newTermStartDate: ISODate("2022-11-24T13:49:50.829Z"),
+    wMajorityWriteAvailabilityDate: ISODate("2022-11-24T13:49:50.840Z")
+  },
+  members: [
+    {
+      _id: 0,
+      name: '10.192.3.58:27017',
+      health: 1,
+      state: 1,
+      stateStr: 'PRIMARY',
+      uptime: 3836,
+      optime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-11-24T13:52:30.000Z"),
+      lastAppliedWallTime: ISODate("2022-11-24T13:52:30.843Z"),
+      lastDurableWallTime: ISODate("2022-11-24T13:52:30.843Z"),
+      syncSourceHost: '',
+      syncSourceId: -1,
+      infoMessage: '',
+      electionTime: Timestamp({ t: 1669297790, i: 2 }),
+      electionDate: ISODate("2022-11-24T13:49:50.000Z"),
+      configVersion: 2,
+      configTerm: 1,
+      self: true,
+      lastHeartbeatMessage: ''
+    }
+  ],
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669297950, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
+      keyId: Long("0")
+    }
+  },
+  operationTime: Timestamp({ t: 1669297950, i: 1 })
+}
+rs0 [direct: primary] test>
+
+Add mongo-1 pod as Secondary Replica
```

Comment:
Another Step-? Even if these aren't "Steps", it seems odd that there's no explanation before getting into the code.

Reply:
its in side big code block i.e. sub step